code/modules/power/apc/apc_main.dm

```diff
@@ -660,8 +644,8 @@
 /obj/machinery/power/apc/proc/overload_lighting()
 	if(!operating || shorted)
 		return
-	if(cell && cell.charge >= 20)
-		cell.use(20)
+	if(cell && cell.charge >= 2e4)
+		cell.use(2e4)
```

Comment:
Some questions. Why was this changed from 20 to 2e4? Why has everything been multipled by 1000?.

Is it because the smes supplies energy in those amounts? Has solar panels output been buffed to keep up with these needs?

Also it would be nice if you could create defines like kilo joules, mega joules and giga joules and attach them to these values for readibility

Reply:
Cells use to have an energy unit that was effectively 1kJ. Now they don't, so using 1 from a cell is 1 joule instead of 1kJ.

Reply:
I might create the defines later if I feel like it. Somehow I missed that part of your comment. You're right and I do think the watt defines could be useful for power consumption defines, so people know whether it's power or energy at a glance.